tests/base.py

```diff
@@ -189,6 +192,20 @@ def tearDownClass(cls):
         settings.UPLOADS_PATH = cls._old_uploads_path
 
 
+class MockUpdateNodeCase(unittest.TestCase):
+    """ Prevents indexing of files when seach.update_node is called.
+    """
+    def setUp(self):
+        super(MockUpdateNodeCase, self).setUp()
+        self.search_patch = mock.patch('website.search.file_util.collect_files',
+                                       side_effect=return_empty_list)
```

Comment:
`side_effect=return_empty_list` can be replaced with `side_effect=lambda: []`, and `return_empty_list` can be removed.